runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -880,6 +887,56 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 		addLoot(event, -1, LootRecordType.EVENT, metadata, items);
 	}
 
+	private boolean handleWildernessLootKeys()
+	{
+		boolean recordedLoot = false;
+
+		for (int containerId : PVP_LOOT_KEY_CONTAINERS)
+		{
+			final Collection<ItemStack> items = toItemStacks(client.getItemContainer(containerId));
+			if (items.isEmpty())
+			{
+				continue;
+			}
+
+			addLoot("Loot Chest", -1, LootRecordType.EVENT, null, items);
```

Comment:
idk if most people would understand "Loot Chest" as a loot source; a quick check with other folks on Discord seems to indicate "Loot key" would be more recognizable (since that's the item turned in to get the loot)
```suggestion
			addLoot("Loot key", -1, LootRecordType.EVENT, null, items);
```